submissions/unabyband/popup-task/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <link rel="stylesheet" href="css/main.css">
+        <title>unabyband pop-up</title>
+    </head>
+    <body>
+        <div class="header">
```

Comment:
Too a lot of divs :) Let's make the markup more semantic friendly with this:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

Reply:
Ok, done! Thanks for your advice.

Reply:
not done fully :) we have a header and a nav inside